webapp/package.json

```diff
@@ -49,10 +49,10 @@
   },
   "proxy": {
     "/api": {
-      "target": "http://localhost:8000"
+      "target": "http://flask:8000"
```

Comment:
does this work on localhost?

Reply:
No, I'm trying to figure out a way to configure this from environment variables. It may require moving the proxy settings from JSON to JS though.

Reply:
is there a way to maybe add another layer so that all of the docker components forward to localhost?

Reply:
You could use a dockerfile, instead of a prebuilt image, to replace localhost using `sed` or `jq`

Reply:
I had the same thought and gave it a shot. Apparently docker treats `/etc/hosts` uniquely though, and it's hard to overwrite. I figured out a much better solution though (see comment below).

Reply:
Sorry I meant replace the string in the package.json but comment below lgtm